cg/cli/workflow/raredisease/base.py

```diff
@@ -95,29 +95,23 @@ def run(
 ) -> None:
     """Run raredisease analysis for given CASE ID."""
```

Comment:
I love this! So clean 🌟 I think this CLI command now is general for all nf analyses so it could be moved to nf_analysis and then add the commands in the different workflows

Reply:
Agreed, implemented